### PR TITLE
regexp-parser: add note of named capture groups

### PR DESCRIPTION
**Pull request by gaborznagy**

### CODE REVIEW

File: Content/Guides/syslog-ng-guide-admin/parser-regexp.htm [hunk 1]
```diff
@@ -17,7 +17,10 @@ <h1 name="parser-regexp">Regular expression (regexp) parser</h1>
...
-        <p>For more information about regular expressions in <MadCap:variable name="General.abbrev" />, see <MadCap:xref href="regular-expressions.htm"><span style="color: #04aada;" class="mcFormatColor">Regular expressions</span></MadCap:xref>.</p><pre>&lt;PRI&gt;&lt;VERSION&gt; &lt;YYYY-MM-DD&gt; &lt;HH-MM-SS&gt; &lt;PROGRAM&gt; &lt;PID&gt; &lt;MSGID&gt; - [key1:value1; key2:value2; ... ]</pre>
+        <div class="Note">
+            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">regexp-parser()</span> can create additional name-value pairs only if &quot;named capture groups&quot; used in the regular expression, for example <span class="Code">(?&lt;test_field&gt;\w+)</span>. For more information see &quot;named capture groups&quot; in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/parser-regexp.htm:21]: "if &quot;named capture groups&quot; used"
missing and "are" before "used"

Comment (tamaskiss00) [reply Content/Guides/syslog-ng-guide-admin/parser-regexp.htm:21]: missing comma after "For more information"

Comment (gaborznagy) [reply Content/Guides/syslog-ng-guide-admin/parser-regexp.htm]: Thanks for the review, fixed.

File: Content/Guides/syslog-ng-guide-admin/parser-regexp.htm [hunk 2]
```diff
@@ -17,7 +17,10 @@ <h1 name="parser-regexp">Regular expression (regexp) parser</h1>
...
-        <p>For more information about regular expressions in <MadCap:variable name="General.abbrev" />, see <MadCap:xref href="regular-expressions.htm"><span style="color: #04aada;" class="mcFormatColor">Regular expressions</span></MadCap:xref>.</p><pre>&lt;PRI&gt;&lt;VERSION&gt; &lt;YYYY-MM-DD&gt; &lt;HH-MM-SS&gt; &lt;PROGRAM&gt; &lt;PID&gt; &lt;MSGID&gt; - [key1:value1; key2:value2; ... ]</pre>
+        <div class="Note">
+            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">regexp-parser()</span> can create additional name-value pairs only if &quot;named capture groups&quot; are used in the regular expression, for example <span class="Code">(?&lt;test_field&gt;\w+)</span>. For more information, see &quot;named capture groups&quot; in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/parser-regexp.htm:21]:
```suggestion
            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">regexp-parser()</span> can create additional name-value pairs only if <span class="Code">&quot;named capture groups&quot;</span> are used in the regular expression, for example <span class="Code">(?&lt;test_field&gt;\w+)</span>. For more information, see <span class="Code">&quot;named capture groups&quot;</span> in <a href="https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC16">PCRE documentation</a>.</p>
```

Comment (gaborznagy) [reply Content/Guides/syslog-ng-guide-admin/parser-regexp.htm:21]: I don't think named capture groups need any code highlighting. My understanding is to use code highlighting for syslog-ng configuration elements, e.g. driver names, option names, etc.

I would even drop the quoting around named capture groups if you agree.

Comment (tamaskiss00) [reply Content/Guides/syslog-ng-guide-admin/parser-regexp.htm:21]: I agree

**Pull request merged**